package.yaml

```diff
@@ -58,3 +58,5 @@ tests:
       - hspec
       - shellwords
       - megaparsec
+    build-tools:
+      - hspec-discover:hspec-discover
```

Comment:
This can be shortened to:
```yaml
build-tools: hspec-discover
```

Hpack allows you to omit the executable name when it is the same as the package name.